packages/aws-cdk/lib/init-templates/v1/app/javascript/package.template.json

```diff
@@ -16,5 +16,8 @@
   },
   "dependencies": {
     "@aws-cdk/core": "%cdk-version%"
+  },
+  "resolutions": {
+    "netmask": "^2.0.1"
```

Comment:
Are those necessary? I mean they'll be shipping with a newer CLI that has a corrected dependency...